library/src/test/java/io/constructor/core/ConstructorIoTrackingTest.kt

```diff
@@ -431,9 +443,8 @@ class ConstructorIoTest {
         mockServer.enqueue(mockResponse)
         val observer = ConstructorIo.trackBrowseResultClickInternal("group_id", "Movies","TIT-REP-1997", 4).test()
         observer.assertError(SocketTimeoutException::class.java)
-        val request = mockServer.takeRequest()
-        val path = "/v2/behavioral_action/browse_result_click?filter_name=group_id&filter_value=Movies&customer_id=TIT-REP-1997&result_position_on_page=4&autocomplete_section=Products&key=copper-key&i=wacko-the-guid&ui=player-three&s=67&c=cioand-2.3.0&_dt=";
-        assert(request.path.startsWith(path))
+        val request = mockServer.takeRequest(10, TimeUnit.SECONDS)
+        assertEquals(null, request)
```

Comment:
throttleBody throttles the request reader and response writer, so it might make sense for the `mockServer.takeRequest(10, TimeUnit.SECONDS)` to return null